Library/Homebrew/cmd/uninstall.rb

```diff
@@ -9,15 +10,41 @@ def uninstall
       ARGV.kegs.each do |keg|
         keg.lock do
           puts "Uninstalling #{keg}... (#{keg.abv})"
+
+          # Find old name and old name symlinks for formula and remove them
+          begin
+            formula = Formulary.from_rack(keg.rack)
```

Comment:
Yes, but how do I know, that it's `foo` linked to `bar`, when I have only `bar`. I need to check the whole Cellar to find that link or somehow find files that are links to `bar`.